test/test_scal_device.cc

```diff
@@ -120,7 +122,7 @@ void test_scal_device_work( Params& params, bool run )
         real_t error = 0;
         int64_t ix = (incx > 0 ? 0 : (-n + 1)*incx);
         for (int64_t i = 0; i < n; ++i) {
-            error = std::max( error, std::abs( (xref[ix] - x[ix]) / xref[ix] ));
```

Comment:
blas::max is a bit more flexible than std::max. It can take > 2 arguments and arguments of different types, e.g.,
`blas::max( 0, m, n )` where 0 is int; m, n are int64. The result is int64, just as the result of `0 + m + n` would be int64.